MediaBrowser.Model/Drawing/ImageDimensions.cs

```diff
@@ -41,5 +68,10 @@ public override string ToString()
                 Width,
                 Height);
         }
+
+        public override int GetHashCode()
+        {
+            return (Height.GetHashCode() * 17) + Width.GetHashCode();
```

Comment:
Why 17?

Reply:
I followed the advice from this SO thread: https://stackoverflow.com/questions/263400/what-is-the-best-algorithm-for-overriding-gethashcode 
I appreciate it's a bit of a magic number - happy to change

Reply:
Adding a comment with that URL seems appropriate here, and maybe a bit of an explanation just in case the URL eventually breaks in the future.